db/index.js

```diff
@@ -0,0 +1,39 @@
+
+const mongoose = require('mongoose');
+
+
+
+mongoose.connect("mongodb://localhost/productDetails", {useNewUrlParser: true, useUnifiedTopology: true});
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  console.log('we are connected!')
+});
+
+let productDetailsSchema = mongoose.Schema({
+  isbn13: String,
+  bookTitle: String,
+  author: String,
+  publisherName: String,
+  publisherLink: String,
+  publicationDate: Date,
```

Comment:
More of a curiosity - has "publicationDate" being set as type "Date" caused any issues returning this as a date rather than string given the "Date" formatting requirements?

Reply:
When I used the find method on the server-side, it returned the date as a string that I could easily convert to the required structure by using Date() on the client-side. The date creates a new Date object.